config/webpack.config.js

```diff
@@ -23,6 +23,8 @@ var resolveModules = [  path.join(__dirname, '..', 'webpack'),
                         'node_modules/',
                      ].concat(pluginUtils.pluginNodeModules(plugins));
 
+var supportedLocales = 'ca|de|en|es|fr|gl|it|ja|ko|pl|pt|ru|sv|zh'
```

Comment:
is there an easy way to maintain this list (instead of hard code?)
also, any idea why the intl js code is so big? which kind of strings are in it to begin with?

Reply:
We could probably `ls ./locale/` and use names of the directories there.

Reply:
perhaps get a list of subfolders in `locale/` and join by `|`?